### PR TITLE
Misc CoreOS Ignition 

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

**Comment by shawndwells** [comment]
@pekramp isolated your changes from the old PR. Once unit tests pass we can squash these commits.

File: ocp4/profiles/moderate.profile [hunk 1]
```diff
@@ -62,6 +62,7 @@ selections:
 
     ### Services
     # sshd
+    - service_sshd_disabled
```
Comment (shawndwells) [inline ocp4/profiles/moderate.profile:65]:
```suggestion
    #- service_sshd_disabled
```

Comment (shawndwells) [reply ocp4/profiles/moderate.profile:65]: Looks like the build is failing by turning SSH off. We can comment out this rule, and then go through the ssh checks re-enabling/disabling as needed in new PRs.

**Comment by JAORMX** [comment]
This sure makes a lot of changes and it's pretty hard to figure out what the issue is. Any chance you can split this into separate PRs?

**Comment by matejak** [comment]
@shawndwells any chance that you can rebase and look at this PR again?

**Comment by mildas** [comment]
**Changes identified:**
Profile moderate on ocp4:
 Rule service_sshd_disabled added to moderate profile.
 Rule harden_sshd_crypto_policy, sshd_set_keepalive, sshd_set_idle_timeout, sshd_disable_rhosts, sshd_limit_user_access removed from moderate profile.

**Recommended tests to execute:**
 build_product ocp4
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-ocp4-ds.xml moderate

**Comment by matejak** [comment]
This PR needs some attention:

- The part where remediations are is likely correct, besides that remediations should go to the `kubernetes` folder as opposed to the `ignition` folder.
- The profile situation has changed - there are now RHEL/RHCOS moderate profiles in addition to the OCP profile, and those selections should go to the other ones now.

**Comment by matejak** [comment]
This PR received no attention since the last comment, so I am closing it.

**Pull request closed without merging**